### ISSUE TITLE
The escape-solidus option should apply to xml-to-json

### ISSUE THREAD

**Issue by ndw** [qt4cg/qtspecs#1347]
Recently, we added an `escape-solidus` option to `fn:serialize()`. That option should apply to `fn:xml-to-json()` as well.

It's not immediately obvious to me if there are other serialize options that should apply as well.

**Comment by ChristianGruen** [comment]
Good point. For `number-formatter`, it’s the other way round: It is available for `fn:xml-to-json`, but not for `fn:serialize`, and for which it would definitely be useful – but it’s probably a bigger step to introduce function item options in the serialization spec.
